src/components/navigation/Navigation.tsx

```diff
@@ -1,19 +1,32 @@
-import { Link } from '@tanstack/react-router'
+import { Link } from "@tanstack/react-router";
+import { AppWindow, Calendar, Student } from "@phosphor-icons/react";
 
 export const Navigation = () => {
+  const links = [
+    { to: "/", label: "Dashboard", icon: AppWindow },
+    { to: "/schedule", label: "Schedule", icon: Calendar },
+    { to: "/students", label: "Students", icon: Student },
+  ] as const;
+
   return (
     <nav>
-      <ul>
-        <li>
-          <Link to="/">Dashboard</Link>
-        </li>
-        <li>
-          <Link to="/schedule">Schedule</Link>
-        </li>
-        <li>
-          <Link to="/students">Students</Link>
-        </li>
+      <ul className="flex flex-col gap-1 mt-4">
+        {links.map((item, idx) => (
+          <Link
+            to={item.to}
+            key={idx}
+            className="text-sm transition-all duration-300 text-center text-slate-500 rounded-md border border-transparent
+            flex items-center gap-1 w-[200px] mx-auto [&.active]:bg-slate-100 [&.active]:border-slate-200 p-2"
```

Comment:
I would add something like `hover:bg-slate-50 focus:bg-slate-100` so we have a button friendly visual WDYT?

https://github.com/dance-app/app/assets/39580007/6fe11bd9-1170-4888-9eb9-de9a6f55ff6b

```suggestion
            className="text-sm transition-all duration-300 text-center text-slate-500 rounded-md border border-transparent
            flex items-center gap-1 w-[200px] mx-auto [&.active]:bg-slate-100 [&.active]:border-slate-200 p-2 hover:bg-slate-50 focus:bg-slate-100"
```